dags/github/neo4j_storage/comments.py

```diff
@@ -33,6 +34,19 @@ def save_review_comment_to_neo4j(review_comment: dict, repository_id: str):
     else:
         user_query = ""
 
+    if reactions_member:
+        reactions_member_query = f"""
+            WITH c
+            UNWIND $reactions_member as reaction_member
+            MERGE (ghu:{Node.GitHubUser.value} {{id: reaction_member.id}})
+                SET ghu += reaction_member, ghu.latestSavedAt = datetime()
+            WITH c, ghu
+            MERGE (c)-[ra:{Relationship.REACTED.value}]->(ghu)
```

Comment:
Is this going to create the `(Comment) -[:REACTED]-> (GitHubUser)`? If so, We want the opposite, meaning github user to react on a comment.